include/bluetooth/mesh/lightness_srv.h

```diff
@@ -140,6 +140,8 @@ struct bt_mesh_lightness_srv {
 	struct bt_mesh_ponoff_srv ponoff;
 	/** Pointer to the model entry in the composition data. */
 	struct bt_mesh_model *lightness_model;
+	/** Pointer to the Setup Server model entry in the composition data. */
+	struct bt_mesh_model *setup_model;
```

Comment:
Should this be named "lightness_setup_model? More similar to what is done in gen_ponoff_srv and gen_plvl_srv, and the line above.